OrbitGl/TimeGraph.cpp

```diff
@@ -914,21 +914,28 @@ void TimeGraph::JumpToNeighborBox(TextBox* from, JumpDirection jump_direction,
   }
   auto function_address = from->GetTimer().m_FunctionAddress;
   auto current_time = from->GetTimer().m_End;
+  auto thread_id = from->GetTimer().m_TID;
   if (jump_direction == JumpDirection::kPrevious) {
-    if (jump_scope == JumpScope::kSameThread) {
+    if (jump_scope == JumpScope::kSameDepth) {
```

Comment:
Can you turn that into a switch (see comment below)?

Reply:
Is ok now? Or you also suggested to do a switch in the jump_direction?

Reply:
No, a second nested switch would harm readability. But regarding the switch on `jump_scope`, you shouldn't provide a `default`-case. When having a `default`-case the compiler won't warn you about non-handled cases (which was the whole purpose of that change.)